service/mantle/ups/UpsServices.xml

```diff
@@ -47,4 +47,80 @@
         </actions>
     </service>
 
+
+    <!-- ========== Label Service ========== -->
+    <service verb="get" noun="OrderShippingLabel">
```

Comment:
The label service must implement the interface, mantle.shipment.CarrierServices.request#ShippingLabels.
I think you need to re think on the label service here according to the interface parameters. Please check and update.